pkg/operator/sync.go

```diff
@@ -204,6 +204,12 @@ func (optr *Operator) syncRenderConfig(_ *renderConfig) error {
 			return err
 		}
 		spec.CloudProviderConfig = cc
+
+		caCert, err := optr.getCAsFromConfigMap("openshift-config", infra.Spec.CloudConfig.Name, "ca-bundle.pem")
+		if err != nil && !apierrors.IsNotFound(err) {
```

Comment:
`apierrors.IsNotFound(err)` catches cases where the configmap doesn't exist, but doesn't cover the case where "ca-bundle.pem" is missing from the configmap.